moped-editor/src/views/projects/projectsMapView/index.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import FallbackComponent from "src/components/FallbackComponent";
+import MapDrawer from "./components/MapDrawer";
+import Page from "src/components/Page";
+import ProjectsMap from "./ProjectsMap";
+import Box from "@mui/material/Box";
+import Paper from "@mui/material/Paper";
+import Stack from "@mui/material/Stack";
+import Typography from "@mui/material/Typography";
+import makeStyles from "@mui/styles/makeStyles";
+import { ErrorBoundary } from "react-error-boundary";
+
+const useStyles = makeStyles((theme) => ({
+  content: {
+    position: "relative",
+    minHeight: "100%",
+    margin: theme.spacing(3),
+    height: "80vh",
+    display: "flex",
+  },
+}));
+
+const ProjectsMapView = () => {
+  const classes = useStyles();
+
+  /* Store map instance to call Mapbox GL methods where needed */
+  const mapRef = React.useRef();
```

Comment:
Rose's comment about arrow fns reminded me to ask about how y'all feel about dot referencing React hooks this way? I think it is kind of a neat way to differentiate React hook vs. custom hooks. I've been seeing this lately in React projects too.